docs/.vuepress/theme/components/ThemeSwitcher.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <label id="switch" class="switch">
+      <input type="checkbox" v-on:change="toggleTheme" id="slider" :checked="isDarkTheme">
+      <span class="slider round"></span>
+  </label>
+</template>
+
+<script>
+const CLASS_THEME_DARK = 'theme-dark';
+const SRC_LOGO_DEFAULT = '/img/handsontable-logo.svg';
+const SRC_LOGO_WHITE = '/img/handsontable-logo-white.svg';
+
+const getDomElement = (selector) => {
+  return document.querySelector(selector);
+};
+
+const toggleDarkThemeClassOnHTML = (htmlDomElement, isDarkTheme) => {
+  if (isDarkTheme) {
+    htmlDomElement.classList.add(CLASS_THEME_DARK);
+
+    return;
+  }
+
+  htmlDomElement.classList.remove(CLASS_THEME_DARK);
+};
+
+const changeLogo = (logoEl, isDarkTheme) => {
+  logoEl.src = isDarkTheme ? SRC_LOGO_WHITE : SRC_LOGO_DEFAULT;
+};
```

Comment:
IMO we can ditch that. Logo is an SVG, so we can manipulate the color with `.theme-dark svg { fill: #fff }`. The only issue is that it's embeded as an `<img>` tag, but we have an override for navbar in `theme/components/Navbar.vue` and we can change it.